SpatialGDK/Source/SpatialGDK/Public/Utils/EngineVersionCheck.h

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Launch/Resources/SpatialVersion.h"
+
+// GDK Version to be updated with SPATIAL_ENGINE_VERSION
+// when breaking changes are made to the engine that requires
+// changes to the GDK to remain compatible
+#define SPATIAL_GDK_VERSION 0
```

Comment:
I know we're all programmers, but I feel version should start with 1 when there's no minor version.

Reply:
please start at -1